File: Python3/프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.py
def solution(a, b):
    numA, numB = a % 2, b % 2
    
    if numA + numB == 2:
        return int(pow(a, 2) + pow(b, 2))
    elif numA + numB == 1:
        return 2 * (a + b)
    else:
        return abs(a - b)

File: Swift/프로그래머스/0/181839. 주사위 게임 1/주사위 게임 1.swift
import Foundation

func solution(_ a:Int, _ b:Int) -> Int {
    var numA = a % 2
    var numB = b % 2
    if numA + numB == 2 {
        return Int(pow(Double(a), 2.0) + pow(Double(b), 2.0))
    } else if numA + numB == 1 {
        return 2 * (a + b)
    } else {
        return abs(a - b)
    }
    return 0
}